Terminal Commands.md

```diff
@@ -48,17 +67,56 @@ Some commands can be found in the `bin` directory.
 `>` - used to store the output of a task in some file(overwrites if file with same name is present) rather than displaying it on the terminal.<br/>
 `>>` - same task as `>` but does not overwrites just appends to the file with the same name.<br/>
 
-`cat` - used to open a file in terminal in read-only format
-
 `|` - piping is used to give the output of a command as input to another command ..for e.g `history | grep "find"` will search for "find" in the output of `history`
 
 `<command> | tee <filename>` - used to show output of `command` on terminal as well as writing the output to a file `filename`.<br/>
 
+`cat` - used to open a file in terminal in read-only format<br>
+`cat file1 > file2` - copies content of file1 one to file2<br>
+Unless you have infinite scrolling turned on (available in Profile Preferences -> Scrolling tab of the terminal), there is a limit to how many lines you can see on the screen.
+
+Piping the output using `less` allows the user to advance through the content by pressing SPACE, move backwards by pressing 'b' and quit using 'q'. Pressing ESC followed by SPACE allows you to scroll down one screen at a time.
+
+`cat file | less`<br>
+
+`history | less` or `cat ~/.bash_history` will let you scroll through previously typed commands.
+
+Piping the outmout using `more` is similar to using `less`, but allows viewing one screen at a time.
+
+- **Trivia:** All commands typed in the terminal are saved in `history` or the `.bash_history` file in the home directory.
+
+##### Aliases <br/>
+An alias is a word assigned to a statement, and acts as a keyboard shortcut.
+
+`alias foo='bar'` - would pass "bar" whenever foo is entered.
+
+This alias lasts as long as the terminal is running. To create a permanent alias, append this line to `~/.bash_profile` or `~/.bash_aliases`
+
 ##### Some common network debugging commands <br/>
 `ifconfig` - when used without any flags, used to display the status of all active network interfaces.<br/>
 
 `iwconfig` - similar to `ifconfig`, but used for wireless network interfaces. <br/>
 
+##### Extracting .tar files <br/>
+`tar -xvzf file.tar.gz` - used to extract the .tar.gz file<br>
+-f: this must be the last flag of the command, and the tar file must be immediately after. It tells tar the name and path of the compressed file.<br>
+-z: tells tar to decompress the archive using gzip<br>
+-x: tar can collect files or extract them. x does the latter.<br>
```

Comment:
I think collect is fine. It might not always be used to compress, just to form a collection. I picked this from an external-source and this seems right.